### PR TITLE
Epirus Substrate Explorer phase 2 application

### PR DESCRIPTION
**Pull request by conor10**
#### Project Abstract

This is an application for phase 2 of the Epirus Substrate Explorer grant which was successfully delivered.

The previous grand application is available [here](https://github.com/w3f/Grants-Program/pull/965).

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [x] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [ x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/grant_guidelines_per_category.md).
- [x ] A BTC, Ethereum (USDT/USDC/DAI) or Polkadot/Kusama (aUSD) address for the payment of the milestones is provided inside the application.
- [x ] The software delivered for this grant will be released under an open-source license specified in the application.
- [x ] The initial PR contains only one commit (squash and force-push if needed).
- [x ] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Review by Micassa93 — APPROVED** [review]

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the application. Could you sign the latest version of our terms and conditions? Do you already have some mock-ups for the planned changes to the UI? I will also share your application already internally.

**Comment by conor10** [comment]
Hi @Noc2, I've signed the updated CLA.

We haven't designed mockups for this next phase - we were keen to get the application in to keep things moving. We can create mockups, however it will take us a day or two to do.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the quick reply. Given the size of the grant it would be great to have some initial mock-ups or other drafts/designs, doesn’t need to be perfect. Let me know in case this would be difficult for you.

**Comment by conor10** [comment]
@Noc2 we've just added mockups and some details on the security front.

**Review by Noc2 — DISMISSED** [review]
Thanks for the update. I’m happy to go ahead with it and will share it with the rest of the team.

**Review by alxs — DISMISSED** [review]
I missed your first grant and wasn't aware of the project - great work!

**Review by gautamdhameja — DISMISSED** [review]

**Comment by conor10** [comment]
Hi, are other reviewers looking at this application? There haven't been any updates in the past 6 days — we're ready to get going once we have approval. 🙂

**Comment by alxs** [comment]
Thanks for pinging us @conor10. I'll share it with the committee again.

**Review by laboon — DISMISSED** [review]

File: applications/epirus_substrate_phase_2.md
Comment (takahser) [inline applications/epirus_substrate_phase_2.md:42]: What compression algorithm are you planning to support here?

**Review by takahser — CHANGES_REQUESTED** [review]

**Comment by mfornos** [comment]
Hi @takahser, 
we just updated the application document in response to your questions.

Summary: 
* We plan to support zip and tgz archive formats
* The public instance is the same but with the added functionality of each milestone

Thank you!

PS: we have noticed that the force push dismissed the previous reviews, hoping it is not a problem. :smile_cat:

**Review by takahser — APPROVED** [review]
Thanks for the update, happy to go ahead with it.

**Review by alxs — APPROVED** [review]

**Comment by conor10** [comment]
@Noc2 and @laboon may we request your re-approval following the updates we've made? We're ready to mobilise once we have those approvals through!

**Review by laboon — APPROVED** [review]

**Review by Noc2 — APPROVED** [review]

**Pull request merged**
